homeassistant/components/govee_light_api/const.py

```diff
@@ -0,0 +1,13 @@
+"""Constants for the Govee Lights - Local API integration."""
+
+from datetime import timedelta
+
+DOMAIN = "govee_light_api"
+MANUFACTURER = "Govee"
+
+CONF_MULTICAST_ADDRESS_DEFAULT = "239.255.255.250"
+CONF_TARGET_PORT_DEFAULT = 4001
+CONF_LISENING_PORT_DEFAULT = 4002
```

Comment:
Typo in const identifier
```suggestion
CONF_LISTENING_PORT_DEFAULT = 4002
```